### DISCUSSION TITLE
Sound for volume changing? :)

### DISCUSSION THREAD

**Discussion by locksoft** [MonitorControl/MonitorControl#544] [Q&A] [answered]
Hi, I've just discovered this app and I already love it (I used SoundControl before).

I wonder if you'll consider the idea to add a sound for volume, as usually macOS does. This helps to set the correct volume without having music or other sound active.

Thanks.

**Comment by waydabber** [comment, marked as answer]
Hi, if the option for this is enabled in Sound Preferences, then there should be a standard click sound - the app checks for this preference and plays the sound for you (there will be a minor tweak in this reagard in the next release so that it plays only on key release).

<img width="780" alt="Screen Shot 2021-09-01 at 8 55 50" src="https://user-images.githubusercontent.com/37590873/131626125-3f0caece-bc16-461e-9582-de2b75825258.png">

Please let me know if it does not work for you! Thanks!

**Reply by locksoft** [reply]
Opps, sorry. Completely missed it, thanks. :)
